docs/sources/flow/reference/config-blocks/remotecfg.md

```diff
@@ -0,0 +1,92 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/config-blocks/remotecfg/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/config-blocks/remotecfg/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/config-blocks/remotecfg/
+- /docs/grafana-cloud/send-data/agent/flow/reference/config-blocks/remotecfg/
+canonical: remotecfgs://grafana.com/docs/agent/latest/flow/reference/config-blocks/remotecfg/
+description: Learn about the remotecfg configuration block
+menuTitle: remotecfg
+title: remotecfg block
+---
+
+# remotecfg block
+
+`remotecfg` is an optional configuration block that enables {{< param "PRODUCT_NAME" >}}
```

Comment:
Does this inherently mean that they cannot define other config blocks in the remote cfg? ie http,logging ect?

Reply:
It cannot contain tracing and logging blocks, but it technically can contain argument and export blocks. I think that we should forbid it as it makes no sense to have them (the controller is marked as isolated)

Reply:
Hmm, that's a good point, should likely be changed.